main/src/ca/uwaterloo/flix/language/phase/typer/PrimitiveEffects.Methods.json

```diff
@@ -1,6 +1,7 @@
 {
   "methods": {
     "java.lang.Boolean::getBoolean": "Env, IO",
+    "java.lang.Float::resolveConstantDesc": "Sys, IO",
```

Comment:
Apparently this does some form of reflection?
https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/Float.html#resolveConstantDesc(java.lang.invoke.MethodHandles.Lookup)